_posts/2022-03-25-deep-learning-for-routing-problems.md

```diff
@@ -0,0 +1,252 @@
+---
+layout: post
+title: Recent Advances in Deep Learning for Routing Problems
+tags: [deep-learning, graph-neural-networks, combinatorial-optimization, travelling-salesperson-problem]
+authors: Joshi, Chaitanya K.; Anand, Rishabh
+---
+
+**TL;DR** Developing neural network-driven solvers for combinatorial optimization problems such as the Travelling Salesperson Problem have seen a surge of academic interest recently. This blogpost presents a **Neural Combinatorial Optimization** pipeline that unifies several recently proposed model architectures and learning paradigms into one single framework. Through the lens of the pipeline, we analyze recent advances in deep learning for routing problems and provide new directions to stimulate future research towards practical impact.
```

Comment:
If you'd like, you could remove the **TL;DR** as this first paragraph will render as a post excerpt in the `blog/` endpoint which will already read like a summary. I leave this up to you though!